sda/overlay.go

```diff
@@ -730,7 +730,7 @@ func (d *defaultProtoIO) Unwrap(msg interface{}) (interface{}, *OverlayMessage,
 	case ProtocolMsg:
 		sdaMsg := inner
 		var err error
-		_, msg, err := network.UnmarshalRegistered(sdaMsg.MsgSlice)
+		_, unmar, err := network.UnmarshalRegistered(sdaMsg.MsgSlice)
```

Comment:
I've named it `protoMsg` then, is it ok? I don't like shadowing a variable and `msg` is already the name of an argument.